ingest/usecase/plugins/orderbook/claimbot/README.md

```diff
@@ -0,0 +1,69 @@
+# Orderbook Claimbot Plugin
+
+The Orderbook Claimbot plugin is a plugin that claims filled order book orders.
+
+It scans all active orders for each order book determining which orders have been filled and need to be claimed. At the moment order is said to be claimable if it is filled 98 percent or more. In order for an order book to be processed to claim its active orders it must be canonical as per SQS definition.
+
+
+Such order book scanning and claiming is achieved by listening for new blocks and core logic is triggered at the end of each new block by calling Claimbot `ProcessEndBlock` method.
+
+## Configuration
+
+### Node
+
+1. Initialize a fresh node with the `osmosisd` binary.
+```bash
+osmosisd init claim-bot --chain-id osmosis-1
+```
+
+2. Get latest snapshot from [here](https://snapshots.osmosis.zone/index.html)
+
+3. Go to `$HOME/.osmosisd/config/app.toml` and set `osmosis-sqs.is-enabled` to true
+
+4. Optionally, turn off any services from `app.toml` and `config.toml` that you don't need
+
+### SQS
+
+In `config.json`, set the plugin to enabled:
+
+```json
+"grpc-ingester":{
+    ...
+    "plugins": [
+        {
+            "name": "orderbook-claimbot-plugin",
+            "enabled": true
+        }
+    ]
+},
+```
+
+Configure the key on a test keyring, and set the following environment variables:
```

Comment:
Sorry for lack of context here, key is named as `local` but then what is meant by `local.info`

Reply:
It's full name of the key on the disk as per my understanding, if you would list contents you would see something like that:

```
ll ~/.osmosisd/keyring-test/
total 16
drwxr-xr-x 2 root root 4096 Oct 25 13:54 ./
drwxr-xr-x 7 root root 4096 Oct 25 13:40 ../
-rw------- 1 root root  537 Oct 25 13:54 abcd....address
-rw------- 1 root root  735 Oct 25 13:54 local.info
```